doc/changelog/05-tactic-language/11981-ltac2-eval-notations.rst

```diff
@@ -0,0 +1,4 @@
+- **Added:**
+  Ltac2 notations for reductions in terms: tacn:: eval @red_expr in @term
```

Comment:
I don't think this is how it gets cited.  @coq/doc-maintainers, how is this supposed to be formatted?

Reply:
Knowing nothing about the RST format, I copied that from ltac.rst, assuming that all .rst files accept the same input format. The result of ltac.rst can be seen here (https://coq.github.io/doc/master/refman/proof-engine/ltac.html#computing-in-a-constr). But the issue with this is that it would link to wrong syntax definitions. I try to find the right ones.

Reply:
@JasonGross : I now looked through the compiled manual for the kind of formatting which is appropriate and then looked at the sources for the RST syntax. As far as I can tell now it is what is standard for inline references to syntax. The format I had before was indeed more a definition display format. I also changed the reference to ltac2_term, but I kept the reference to the ltac reduction definition, since it should be the same.

The examples I chose are the inline syntax definitions in the first text paragraph under (https://coq.github.io/doc/master/refman/proof-engine/ltac.html#proving-a-subgoal-as-a-separate-lemma).

Reply:
I confirm that this is good now.